tests/test_file.py

```diff
@@ -56,6 +45,105 @@ def test_file_put(self):
         )
         self.assertIn('timestamp', response[2])
 
+    def test_file_head(self):
+        file_uuid = "1245b18c-b36b-4ebb-8220-0e0c6f4a87a4"
+        timestamp = "2017-06-16T19:36:04.240704Z"
+
+        url = urllib.parse.urlunparse((
+            "",
+            "",
+            "/v1/files/" + file_uuid,
+            "",
+            urllib.parse.urlencode(
+                (("replica", "aws"),
+                 ("timestamp", timestamp)),
+                doseq=True,
+            ),
+            "",
+        ))
```

Comment:
Is there a way to use kwargs here to make it more explicit what the different args refer to (query_string, host/netloc, etc)?